### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by PatrickMassot** [nickcoutsos/keymap-editor#208] [General]
Thank you very much for your work on this! I soldered my first wireless keyboard yesterday and your editor really made it a lot easier to go over the QMK to ZMK bump. I really wish this could be mentioned in the ZMK documentation. I was lucky that someone took pity on me and told me about this important ZMK secret!

**Comment by nickcoutsos** [comment]
Thanks for the kind words! I've seen a few youtube videos spreading the word about the keymap editor, and I know some keyboard vendors will point customers to it as well so it's always nice to know there's good word-of-mouth out there :)

Congrats on building the keyboard!
